build/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```

Comment:
There's really nothing less ugly?

```
DIR=$(realpath $(dirname $(dirname $0)))
```
?

Reply:
Nope, unfortunately not, see:

https://www.ostricher.com/2014/10/the-right-way-to-get-the-directory-of-a-bash-script/
https://stackoverflow.com/questions/59895/get-the-source-directory-of-a-bash-script-from-within-the-script-itself

Reply:
It's even worse (or not really possible at all) in Posix sh

https://stackoverflow.com/questions/29832037/how-to-get-script-directory-in-posix-sh#29834779